### PR TITLE
Add issue and pull request template

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Related issues/PRs | https://github.com/sulu/sulu/pull/3981
| License | MIT

#### What's in this PR?

Add issue and pull request template.

#### Why?

To add a checkbox for sulu/sulu PR.

#### To Do

- ~~[x] Create sulu/sulu PR~~
- [x] Add breaking changes to UPGRADE.md

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+#### Steps to Reproduce
+
+What are the steps to reproduce this bug? Please add code examples,
+screenshots or links to GitHub repositories that reproduce the problem.
+
+#### Possible Solutions
+
+If you have already ideas how to solve the issue, add them here.
+(remove this section if not needed)
+
```
Comment (danrot) [inline .github/ISSUE_TEMPLATE.md]: Only one new line at the end

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+#### BC Breaks/Deprecations
+
+Describe BC breaks/deprecations here. (remove this section if not needed)
+
+#### To Do
+
+- [ ] Create sulu/sulu PR
+- [ ] Add breaking changes to UPGRADE.md
+
```
Comment (danrot) [inline .github/PULL_REQUEST_TEMPLATE.md]: Only one new line at the end

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+$foo->doSomething();
+~~~
+
+#### BC Breaks/Deprecations
+
+Describe BC breaks/deprecations here. (remove this section if not needed)
+
+#### To Do
+
+- [ ] Create sulu/sulu PR
```
Comment (danrot) [inline .github/PULL_REQUEST_TEMPLATE.md]: Maybe something like `Move changes to sulu/sulu develop branch`, where one part of that is a link to the specific folder in github? Just to be very clear what is meant here.

**Review by danrot — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (alexander-schranz) [reply .github/ISSUE_TEMPLATE.md]: its only one new line at the end? or do you mean no new lines?

**Review by danrot — APPROVED** [review]

**Pull request merged**